go.mod

```diff
@@ -8,17 +8,34 @@ require (
 )
 
 require (
+	github.com/Masterminds/goutils v1.1.0 // indirect
+	github.com/Masterminds/semver v1.5.0 // indirect
+	github.com/Masterminds/sprig v2.22.0+incompatible
+	github.com/araddon/dateparse v0.0.0-20200409225146-d820a6159ab1
+	github.com/ghodss/yaml v1.0.0
```

Comment:
double cvheck we don't already have a yaml package

Reply:
This one seems correct. There is another yaml lib at the bottom which should be removed.